### PR TITLE
tests, les, eth: remove mentions of deprecated Morden testnet

### PR DESCRIPTION
**Pull request by Zachinquarantine**

### CODE REVIEW

**Comment by Zachinquarantine** [comment]
I don't see what test file I'm supposed to delete...

File: tests/difficulty_test.go [hunk 1]
```diff
@@ -52,11 +52,8 @@ func TestDifficulty(t *testing.T) {
 
 	// files are 2 years old, contains strange values
 	dt.skipLoad("difficultyCustomHomestead\\.json")
-	dt.skipLoad("difficultyMorden\\.json")
-	dt.skipLoad("difficultyOlimpic\\.json")
 
 	dt.config("Ropsten", *params.RopstenChainConfig)
-	dt.config("Morden", *params.RopstenChainConfig)
```
Comment (holiman) [inline tests/difficulty_test.go:59]: You can't skip-load these, if they still exist in the ehtereum/tests subrepo from which we load the tests. Are they files gone?

Comment (holiman) [reply tests/difficulty_test.go:59]: https://github.com/ethereum/tests/blob/develop/BasicTests/difficultyMorden.json

**Comment by Zachinquarantine** [comment]
blocked until the next version of ethereum/tests gets published

**Comment by Zachinquarantine** [comment]
Huh, I just realized that [some new tests were added](https://github.com/ethereum/tests/commit/5577afde5963362e80c6a83627be611642bf825d) and [some old tests were deleted](https://github.com/ethereum/tests/commit/e5ddca84840670c953b1f1e42c51c058c7e642b8). Will this break Geth at all?

**Comment by Zachinquarantine** [comment]
Appveyor fails with:
```--- FAIL: TestState (0.58s)
    --- FAIL: TestState/stTransactionTest/HighGasPrice.json (0.07s)
        --- FAIL: TestState/stTransactionTest/HighGasPrice.json/Frontier/0/snap (0.04s)
            state_test.go:120: snapshot [0x7e18cda82e88b51f339192016b7f3e803051e1e115260efb25f8602b894226ec] missing
            state_test.go:135: no EVM operation logs generated
        --- FAIL: TestState/stTransactionTest/HighGasPrice.json/Homestead/0/snap (0.00s)
            state_test.go:120: snapshot [0x7e18cda82e88b51f339192016b7f3e803051e1e115260efb25f8602b894226ec] missing
            state_test.go:135: no EVM operation logs generated
        --- FAIL: TestState/stTransactionTest/HighGasPrice.json/EIP150/0/snap (0.00s)
            state_test.go:120: snapshot [0x7e18cda82e88b51f339192016b7f3e803051e1e115260efb25f8602b894226ec] missing
            state_test.go:135: no EVM operation logs generated```

**Comment by Zachinquarantine** [comment]
bump

**Review by MariusVanDerWijden — APPROVED** [review]
SGTM

**Review by holiman — APPROVED** [review]
LGTM

**Pull request merged**